dxr/mime.py

```diff
@@ -4,17 +4,18 @@
 def icon(path):
     """Return the basename (no extension) of the icon file to use for a path."""
     root, ext = splitext(path)
-    return ext_map.get(ext[1:], 'unknown')
+    return ext_map.get(ext[1:].lower(), 'unknown')
 
 
 def is_text(data):
     # Simple stupid test that apparently works rather well :)
     return '\0' not in data
 
+
 def is_image(path):
     """Determine whether the path is an image."""
```

Comment:
For example, we might have `is_image()` and `is_image_with_indexable_text()` and just call those from various points below rather than hard-coding 'svg' specifically. Then we make the meaning super-clear at all the callsites and, as a bonus, have a hook for any other image-but-text formats that crop up.